cabal.project

```diff
@@ -1,6 +1,12 @@
 packages: .
 
--- source-repository-package
---   type: git
---   location: https://github.com/composewell/streamly
---   tag: master
+source-repository-package
+   type: git
+   location: https://github.com/composewell/streamly.git
+   tag: e9d044d8e5be37e8a1c93d1db5ceeb9f5cdd54f8
+
+source-repository-package
+   type: git
+   location: https://github.com/composewell/streamly.git
+   tag: e9d044d8e5be37e8a1c93d1db5ceeb9f5cdd54f8
+   subdir: core
```

Comment:
We do not keep git repo entries in cabal.project because of a cabal/hls bug that leaves the local repo in detached head mode.